zerver/lib/actions.py

```diff
@@ -1630,10 +1725,12 @@ def do_change_is_admin(user_profile, value, permission='administer'):
         send_event(event, active_user_ids(user_profile.realm))
 
 def do_change_bot_type(user_profile, value):
+    # type: (UserProfile, int) -> None
     user_profile.bot_type = value
     user_profile.save(update_fields=["bot_type"])
 
 def do_make_stream_public(user_profile, realm, stream_name):
+    # type: (UserProfile, Realm, text_type) -> Dict[None, None]
```

Comment:
Same comment here that we should get rid of all the `return {}`s that don't sometimes return some other dict.